screen.js

```diff
@@ -4,15 +4,119 @@ const BUTTON_GAP = 10;
 
 /** A base class for a screen: analogous to a page, scene, etc. */
 class Screen {
-  constructor(name, description) {
+  static BgHorizontalAlign = {
+    LEFT: 'Left',
+    CENTER: 'Center',
+    RIGHT: 'Right',
+  }
+  
+  static BgVerticalAlign = {
+    TOP: 'Top',
+    CENTER: 'Center',
+    BOTTOM: 'Bottom',
+  }
+
+  constructor(
+		name,
+		description,
+		bgColor = null,
+		bgImage = null,
+		bgHorizontalAlign = Screen.BgHorizontalAlign.CENTER,
+		bgVerticalAlign = Screen.BgVerticalAlign.CENTER
+	) {
     this._returnToPreviousScreen = false;
 	  this.name = name;
+		this.bgColor = bgColor;
+    this.bgImage = bgImage;
+    this.bgHorizontalAlign = bgHorizontalAlign;
+    this.bgVerticalAlign = bgVerticalAlign;
+    this.formattedBgImage = null;
 	  this.description = description;
-  }
+  } 
 
   /** Initialization of elements/components/states in the screen */
   init() {}
 
+  /** Draw the background to the screen */
+	drawBackground() {
+		if (this.bgColor && this.bgImage) {
+			print("Error: Both a background color and image were specified. Defaulting to color.");
```

Comment:
Seems less like an error than just a documented behaviour

Reply:
Documented behaviour and removed error.